product_code/authentication_service/impl/src/main/resources/prod-application.conf

```diff
@@ -8,6 +8,13 @@ play {
   http.secret.key = ${APPLICATION_SECRET}
 }
 
+uc4.secrets {
+    master = ${UC4_MASTER_SECRET}
+    salts {
+        kafka = ${UC4_KAFKA_SALT}
```

Comment:
Why is there a salt group?

Reply:
To make the config file a bit more readable. The group is used to provide a separator between the secrets and the salts. Of course it looks weird, because we only have one salt atm and we may change this if we only need one salt.